docs/examples/example_satellite_observations.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>I think it would be cleanest to pick just one of these options, and then showcase that one. Commented code may look messy in the documentation. I would go with the third option because this is the one that will work from anywhere. </p><p> </p><p>As for the option that uses pangeo data: Maybe you can provide a similar notebook to the pangeo gallery and plug gcm-filters there? :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>